src/server/console/logind_console_services.cpp

```diff
@@ -189,7 +192,7 @@ std::string object_path_for_current_session(LogindSeat* seat_proxy)
 }
 
 std::unique_ptr<GDBusConnection, decltype(&g_object_unref)>
-connect_to_system_bus(mir::GLibMainLoop& ml)
+connect_to_system_bus(mir::GLibMainLoop &ml)
```

Comment:
There are a lot of spurious, whitespace changes to this file. A lot of them away from project style